<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-150</y>
     <width>1080</width>
     <height>864</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/bbb111.jpg</pixmap>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>541</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="topMargin">
     <number>40</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="name_label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 105, 214,0.7);

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>      имя:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #ec5b68;
 background-color: rgba(19,9,10, 0.6); ;
font: 700 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="passwd_label">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(3, 105, 214,0.7);

font: 400 11pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>    пароль:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passwd_lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #d55a2c;
 background-color: rgba(19,9,10, 0.6); ;
font: 700 9pt &quot;Segoe Print&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="go_Button">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(163,84,143, 0.8);
color: #f3f2f8;
border-radius:8px;
font: 400 11pt &quot;Yu Gothic UI Light&quot;;</string>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_Button">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(177,96,105, 0.8);
color: #f3f2f8;
border-radius:8px;
font: 400 11pt &quot;Yu Gothic UI Light&quot;;</string>
        </property>
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
